packages/backend/src/accounting/tigerbeetle/service.test.ts

```diff
@@ -232,7 +238,11 @@ describe('TigerBeetle Accounting Service', (): void => {
         transferRef: '00000000-0000-0000-0000-000000000000'
       })
       expect(transferDebit.timestamp).toBeGreaterThan(0)
-      expect(transferDebit.expiresAt).toBeUndefined()
+
+      const debitExpiresAt = transferDebit.expiresAt
+        ? transferDebit.expiresAt.getTime()
+        : now
```

Comment:
related to comment below, maybe we can make a pending transfer in this test and check that `expiresAt` is correctly set